py/objarray.c

```diff
@@ -596,6 +602,13 @@ const mp_obj_type_t mp_type_bytearray = {
 #endif
 
 #if MICROPY_PY_BUILTINS_MEMORYVIEW
+#if MICROPY_PY_BUILTINS_BYTES_HEX
+STATIC const mp_rom_map_elem_t memoryview_locals_table[] = {
+    { MP_ROM_QSTR(MP_QSTR_hex), MP_ROM_PTR(&mp_obj_bytes_hex_as_str_obj) },
```

Comment:
couldn't this use a 1-element slice from `array_bytearray_str_bytes_locals_table` instead, to save this locals table?

Reply:
It could. Saves 8 bytes!